applications/precision/adaptive/README.md

```
@@ -0,0 +1,69 @@
+View them with any image viewer, or convert to PNG:
+
+```bash
+# ImageMagick
+convert mandelbrot_efloat.ppm mandelbrot_efloat.png
+# or Python / Pillow
+python3 -c "from PIL import Image; Image.open('mandelbrot_efloat.ppm').save('mandelbrot_efloat.png')"
+```
+
+### Expected output
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)